src/status_im2/navigation/core.cljs

```diff
@@ -270,6 +278,12 @@
     (fn [] (gesture/gesture-handler-root-hoc views/popover-comp))
     (fn [] views/popover-comp))
 
+   (navigation/register-component
+    "toasts"
+    ;; DON'T wrap this with gesture-handler-root-hoc, it makes it unable to click through toasts
+    (fn [] views/toasts-comp)
```

Comment:
if you don't need gesture you can pass only one function, don't need to duplicate

Reply:
hm im not sure if js/undefined is needed?